xcube/util/dask.py

```diff
@@ -135,6 +139,67 @@ def get_chunk_slices(chunk_sizes: Sequence[int]) -> Iterable[slice]:
         yield slice(start, stop)
 
 
+def new_cluster(
+    provider: str = 'coiled',
+    name: Optional[str] = None,
+    software: Optional[str] = None,
+    n_workers: int = 4,
+    resource_tags: Optional[Dict[str, str]] = None,
+    account: str = 'bc',
+    **kwargs,
+) -> distributed.deploy.Cluster:
+    if resource_tags is None:
+        resource_tags = {
+            'cost-center': 'unknown',
+            'environment': 'dev',
+            'creator': 'auto',
+            'purpose': 'xcube dask cluster',
+        }
+    # TODO: there should be a way to set account and tags without passing them
+    # as function arguments. Reading them from environment variables, perhaps.
+
+    if provider == 'coiled':
+        import coiled
```

Comment:
Catch `ImportError` and re-raise with message saying that provide "coild" requires package `coiled` to be installed.